exercises/word-count/test/word_count_test.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:test/test.dart';
+import 'package:word_count/word_count.dart';
+
+void main() {
+  final mywordcounter = new WordCount();
+  group("WordCount", () {
+    test("count one word", () {
```

Comment:
Add a space between mywordcounter and group.

In order to following the Dart Style Guide, rename mywordcounter to [camelCase syntax](https://www.dartlang.org/guides/language/effective-dart/style#do-name-other-identifiers-using-lowercamelcase).